src/main/java/com/minecolonies/coremod/items/ItemScepterPermission.java

```diff
@@ -0,0 +1,195 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import com.minecolonies.coremod.network.messages.ChangeFreeToInteractBlockMessage;
+import com.minecolonies.coremod.util.LanguageHandler;
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Permission scepter. used to add free to interact blocks or positions to the colonies permission list
+ */
+public class ItemScepterPermission extends AbstractItemMinecolonies
+{
+    /**
+     */
+    private static final String TAG_ITEM_MODE = "scepterMode";
+
+    private static final String TAG_VALUE_MODE_BLOCK = "modeBlock";
+    private static final String TAG_VALUE_MODE_LOCATION = "modeLocation";
+
+    /**
+     */
+    public ItemScepterPermission()
+    {
+        super("scepterPermission");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)